phoenix-core/src/main/java/org/apache/phoenix/index/IndexMaintainer.java

```diff
@@ -954,13 +981,14 @@ public int getEstimatedByteSize() {
         size += dataEmptyKeyValueCF.length + WritableUtils.getVIntSize(dataEmptyKeyValueCF.length);
         size += emptyKeyValueCFPtr.getLength() + WritableUtils.getVIntSize(emptyKeyValueCFPtr.getLength());
         size += WritableUtils.getVIntSize(nDataCFs+1);
+        //TODO figure out how to estimate size of indexedExpressions
```

Comment:
There's not a great way to do this. Maybe just have a constant size and multiply by the number of functional expressions.